### PR TITLE
[Merged by Bors] - chore: Reduce imports to `Data.Finset.Image`

### PR DESCRIPTION
**Pull request by YaelDillies**
---

- [x] depends on: #12828

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — COMMENTED** [review]
You're not actually reducing any imports in Data.Finset.Image, as far as I can tell. Maybe it's worth landing anyway, but in that case please change the PR title.

**Review by Ruben-VandeVelde — COMMENTED** [review]
That makes more sense.

maintainer merge

File: Mathlib/Data/Finset/Card.lean [hunk 1]
```diff
@@ -26,6 +26,9 @@ This defines the cardinality of a `Finset` and provides induction principles for
 * `Finset.Nonempty.strong_induction`
 -/
 
+-- TODO
+-- assert_not_exists OrderedCommMonoid
+-- assert_not_exists Ring
```
Comment (YaelDillies) [inline Mathlib/Data/Finset/Card.lean:31]: FYI, this TODO is waiting on #12836

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Comment by riccardobrasca** [comment]
I guess this needs a `merge master`.

bors d+

File: Mathlib/Data/Finset/Card.lean (hunk 1 above)
Comment (riccardobrasca) [reply Mathlib/Data/Finset/Card.lean:31]: Can you fix it now?

Comment (YaelDillies) [reply Mathlib/Data/Finset/Card.lean:31]: Actually no I can't. I was wrong. Removing `Ring` depends on #12845 instead. Removing `OrderedCommMonoid` seems very hard right now.

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**